fake-snippets-api/routes/api/snippets/add_star.ts

```diff
@@ -24,14 +24,6 @@ export default withRouteSpec({
     })
   }
 
-  // Check if already starred
-  if (ctx.db.hasStarred(ctx.auth.account_id, snippet_id)) {
-    return ctx.error(400, {
-      error_code: "already_starred",
-      message: "You have already starred this snippet",
-    })
-  }
```

Comment:
why is this removed?

Reply:
ig there was no need to include an already starred condition as we were handling it in the `/remove_star` route. will revert these changes.